prefect_dbt/cloud/jobs.py

```diff
@@ -113,6 +113,8 @@ def trigger_dbt_cloud_job_run_flow():
         raise DbtCloudJobRunTriggerFailed(extract_user_message(ex)) from ex
 
     run_data = response.json()["data"]
+    if run_data.get("id") is None:
+        raise RuntimeError("Unable to determine run ID for triggered job.")
```

Comment:
I'm not sure if we should raise an error from here if we can't find an ID, because the consumer may not care that there isn't an ID in the response. We could conditionally log only if `id` and `project_id` are in `run_data` to avoid a potential `KeyError`.